packages/mobile/src/components/rate-cta-drawer/RateCtaDrawer.tsx

```diff
@@ -52,6 +52,9 @@ const useStyles = makeStyles(({ palette, typography, spacing }) => ({
     width: 18,
     marginRight: spacing(2)
   },
+  buttonText: {
+    fontSize: typography.fontSize.large
```

Comment:
Is there a way to configure the button using props and defaults? Just hoping to prevent one off styling, like we could use the size prop on button maybe? That makes the whole button bigger as well though. Happy to have a convo about this

Reply:
yea i agree. there were a few things about the drawer that needed one-off styling to conform with the designs. I feel like a lot of this will get tackled when we make the big design system push